frontend/src/metabase/actions/containers/ActionCreator/tests/ActionCreator-ImplicitActions.unit.spec.ts

```diff
@@ -0,0 +1,80 @@
+import nock from "nock";
+
+import { screen, queryIcon } from "__support__/ui";
+
+import {
+  createMockActionParameter,
+  createMockImplicitQueryAction,
+} from "metabase-types/api/mocks";
+
+import { setup as baseSetup, SetupOpts } from "./common";
+
+async function setup({
+  action = createMockImplicitQueryAction(),
+  ...opts
+}: SetupOpts = {}) {
+  await baseSetup({ action, ...opts });
+  return { action };
+}
+
+describe("ActionCreator > Implicit Actions", () => {
+  afterEach(() => {
+    nock.cleanAll();
+  });
+
+  it("renders correctly", async () => {
+    const { action } = await setup();
+
+    expect(screen.getByText(action.name)).toBeInTheDocument();
+    expect(screen.getByText("Auto tracking schema")).toBeInTheDocument();
+    expect(screen.getByRole("button", { name: "Cancel" })).toBeInTheDocument();
+
+    expect(screen.queryByText(/New action/i)).not.toBeInTheDocument();
+    expect(
+      screen.queryByTestId("mock-native-query-editor"),
+    ).not.toBeInTheDocument();
+  });
+
+  it("renders parameters", async () => {
+    await setup({
+      action: createMockImplicitQueryAction({
+        parameters: [createMockActionParameter({ name: "FooBar" })],
+      }),
+    });
+
+    expect(screen.getByText("FooBar")).toBeInTheDocument();
+  });
+
+  it("allows only form settings changes", async () => {
```

Comment:
I'm a little confused with the test name. It should allow something, but I only see you assert 3 things are not allowed or are invisible. I would expect at least one assertion on **form settings change** in some way.